packages/gafl-webapp-service/src/processors/concession-helper.js

```diff
@@ -74,6 +74,11 @@ export const removeDisabled = permission => {
   }
 }
 
+export const getAgeConcession = permission =>
+  permission.concessions
+    ? permission.concessions.find(c => c.type !== CONCESSION.DISABLED)
```

Comment:
Very minor and you can leave this as is. But it might be better to be more explicit and say if it's CONCESSION.JUNIOR or CONCESSION.SENIOR

Reply:
I had that and then changed it. I couldn't make up my mind! But I do agree with you. I'll change it back

Reply:
Or, more succinctly, do
```suggestion
    ? permission.concessions.find(c => [CONCESSION.JUNIOR, CONCESSION.SENIOR].includes(c.type))
```

Reply:
Yes, you're right Phil, that is the best way